config/esp32/components/chip/CMakeLists.txt

```diff
@@ -30,12 +30,7 @@ if(NOT CHIP_ROOT)
 endif()
 
 idf_component_register(SRCS chip.c chip.cpp
-                       PRIV_REQUIRES freertos lwip bt mdns mbedtls)
-
-# Override some build specifications
-set_property(TARGET ${COMPONENT_LIB} PROPERTY CXX_STANDARD 14)
-set(CMAKE_BUILD_TYPE MinSizeRel)
-target_compile_options(${COMPONENT_LIB} PRIVATE "-DLWIP_IPV6_SCOPES=0" "-DCHIP_HAVE_CONFIG_H")
```

Comment:
@sweetymhaiske Where are the corresponding new additions?

Reply:
Yes, that's how it is done for Makefile too.